docker-compose.prod.yaml

```diff
@@ -76,30 +76,35 @@ services:
       # These values are unique to Sepolia Testnet
       # They'll need to be manually changed in case the testnet is reset
       # To add robustness: parse the `deployments/kakarot-sepolia` folder in `kkrt-labs/kakarot` repo
-      - KAKAROT_ADDRESS=0x70c14f7fe5968975cff5cc4614c9d9a8cbb9051da19a28990772e3b55c23328
+      - KAKAROT_ADDRESS=0x11c5faab8a76b3caff6e243b8d13059a7fb723a0ca12bbaadde95fb9e501bda
       - UNINITIALIZED_ACCOUNT_CLASS_HASH=0x600f6862938312a05a0cfecba0dcaf37693efc9e4075a6adfb62e196022678e
       - ACCOUNT_CONTRACT_CLASS_HASH=0x490cccb64e3917ecf0a80a59d0c3e449766f745c1d6db54e0050f89eb59aba1
       - MAX_FELTS_IN_CALLDATA=30000
-      - RETRY_TX_INTERVAL=10
+      - RETRY_TX_INTERVAL=2
     restart: on-failure
     volumes:
       # Mount the indexer code
-      - indexer_code:/usr/src/indexer
+      - indexer_code_prod:/usr/src/indexer
     depends_on:
       starknet:
         condition: service_started
     networks:
       - internal_prod
 
   apibara-dna:
-    extends:
-      file: docker-compose.yaml
-      service: apibara-dna
+    image: quay.io/apibara/starknet:1.4.1
     command:
       - start
       - --rpc=http://starknet:6060
       - --wait-for-rpc
       - --data=/data
+    ports:
+      - 7171:7171
+    volumes:
+      - apibara_prod:/data
+    networks:
+      - internal_prod
+    restart: on-failure
```

Comment:
any reason for removing the extend? It adds a lot of duplication

Reply:
yes it doesn't work without removing the extend